index.html

```diff
@@ -33,52 +33,83 @@
+				</div>
+	    </div>
+	    <div class="item" id="seating-bg">
+	      <div class="carousel-caption">
+	        <!-- (h4>{Eat Local.})+p>lorem27 -->
+					<h2>Eat Local.</h2>
+					<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Provident id unde libero exercitationem eum suscipit, iste, reiciendis beatae culpa placeat et aperiam rem molestias ratione, ea. Cumque.</p>
+					<!-- button.btn.btn-lg.btn-warning>{Learn more} -->
+					<button class="btn btn-lg btn-warning">Learn more</button>
+	      </div>
+	    </div>
+			<div class="item" id="farm-bg">
+	      <div class="carousel-caption">
+					<h2>From farm to table.</h2>
+					<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Provident id unde libero exercitationem eum suscipit, iste, reiciendis beatae culpa placeat et aperiam rem molestias ratione, ea. Cumque.</p>
+					<button class="btn btn-lg btn-warning">Browse gallery</button>
+	      </div>
+	    </div>
+	  </div>
+
+	  <!-- Controls -->
+	  <a class="left carousel-control" href="#carousel-example-generic" role="button" data-slide="prev">
+	    <span class="glyphicon glyphicon-chevron-left" aria-hidden="true"></span>
+	    <span class="sr-only">Previous</span>
+	  </a>
+	  <a class="right carousel-control" href="#carousel-example-generic" role="button" data-slide="next">
+	    <span class="glyphicon glyphicon-chevron-right" aria-hidden="true"></span>
+	    <span class="sr-only">Next</span>
+	  </a>
+	</div>
+	<!-- carousel ends -->
+	<!-- story section -->
+	<!-- .container>.row>(.col-sm-6>(h1>{Our Story})+h3>{One Man. One Dream.})+.col-sm-6>(h1.year-text>{1989})+(p>lorem)*4 -->
+	<div class="container about">
+		<div class="row">
+			<div class="col-sm-6 story">
+				<h1>Our Story</h1>
+				<h3>One Man. One Dream.</h3>
+			</div>
+			<div class="col-sm-6 year">
+				<h1 class="year-text">1989</h1>
+				<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Autem nisi officia totam, magnam veniam dolore amet voluptatem nostrum assumenda vel ad voluptate, deleniti tenetur cupiditate, ab, eos soluta iure adipisci?</p>
+				<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Autem nisi officia totam, magnam veniam dolore amet voluptatem nostrum assumenda vel ad voluptate, deleniti tenetur cupiditate, ab, eos soluta iure adipisci?</p>
+				<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Autem nisi officia totam, magnam veniam dolore amet voluptatem nostrum assumenda vel ad voluptate, deleniti tenetur cupiditate, ab, eos soluta iure adipisci?</p>
+				<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Autem nisi officia totam, magnam veniam dolore amet voluptatem nostrum assumenda vel ad voluptate, deleniti tenetur cupiditate, ab, eos soluta iure adipisci?</p>
+			</div>
+		</div>
+	</div>
+	<!-- story section ends -->
+	<!-- review section -->
+	<div class="container-fluid review">
+		<div class="row">
+			<div class="wrapper">
```

Comment:
Normally I tell students to make this a column small 8 with the grid and offset it using be there grid system, but this is an interesting approach.

Reply:
So what would be the pros? just lest code? more responsive? should i change it?